Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
```

Comment:
### [rb-inotify](https://github.com/guard/rb-inotify)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.8.8 → 0.10.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/rb-inotify/0.8.8/0.10.1)_]





<details>
<summary>Commits</summary>

A change of **102** commits. See the full changes on [the compare page](https://github.com/guard/rb-inotify/compare/v0.8.8...v0.10.1).

These are the first 10 commits:
- (3488a84) [Use more durable FFI::Library::LIBC constant.](https://github.com/guard/rb-inotify/commit/3488a844b55d092f95541757a0110fee31512108)
- (3255dca) [Merge pull request #20 from mbj/master](https://github.com/guard/rb-inotify/commit/3255dcaece75ef515a8e3ff1c51efd9bac355e52)
- (f25e18e) [Expose Notifier.read_events.](https://github.com/guard/rb-inotify/commit/f25e18e39b542de724542fdfffccdd8b6abc82fa)
- (aca1714) [Update lib/rb-inotify/notifier.rb](https://github.com/guard/rb-inotify/commit/aca171497f492b3846263401ba452210887480c1)
- (28b1f51) [Merge pull request #22 from geoffyoungs/patch-1](https://github.com/guard/rb-inotify/commit/28b1f5162cbdf03416e70fe139db38f5c2115d30)
- (c90466d) [Merge branch 'master' of github.com:nex3/rb-inotify](https://github.com/guard/rb-inotify/commit/c90466dcd0b9f0af1ba804fc20800fa1809482e2)
- (c9ed1f1) [Version bump to 0.9.0](https://github.com/guard/rb-inotify/commit/c9ed1f19819eaa20f67d18778cf7be188ad5f8f3)
- (6a8f19c) [Regenerate gemspec for version 0.9.0](https://github.com/guard/rb-inotify/commit/6a8f19cf31cb84783ddfb99aebe4cb16f34e5219)
- (536d1c1) [Rename conflict variable with the following block](https://github.com/guard/rb-inotify/commit/536d1c1acec33f299438a96a7b2c8d382a88a63c)
- (f74b12f) [Merge pull request #24 from kachick/improve-remove_warnings](https://github.com/guard/rb-inotify/commit/f74b12f7e2433b763dfd7967fa4869a95f8ab1eb)


</details>